docs/ref/models/fields.txt

```diff
@@ -426,6 +426,26 @@ precedence when creating instances in Python code. ``db_default`` will still be
 set at the database level and will be used when inserting rows outside of the
 ORM or when adding a new field in a migration.
 
+If a field is configured with db_default but lacks a default, accessing the field
+on an unsaved model instance, where no value has been explicitly assigned,
+will return a DatabaseDefault object::
+
+    from django.db import models
+    from django.db.models.functions import Now
+
```

Comment:
```suggestion


```
(As a guess) but running blacken-docs should sort it

Reply:
@sarahboyce It passed the test now, But does the changes seem appropriate now?